docs/source/usage/plugins/openPMD.rst

```diff
@@ -134,6 +134,12 @@ PIConGPU command line option          description
 ``--checkpoint.openPMD.jsonRestart``  Set backend-specific parameters for openPMD backends in JSON format for restarting from a checkpoint.
 ``--openPMD.dataPreparationStrategy`` Strategy for preparation of particle data ('doubleBuffer' or 'mappedMemory'). Aliases 'adios' and 'hdf5' may be used respectively.
 ``--openPMD.toml``                    Alternatively configure the openPMD plugin via a TOML file (see below).
+``--openPMD.particleIOChunkSize``     Particle data will be written in chunks of the given size. unit: MiB
+                                      The memory footprint on the host side is reduced compared to writing all particles with one write call.
+                                      Currently the memory footprint is only reduced bp5 is used, HDF5 and bp4 does not support partial data dump.
```

Comment:
This is not a sentence. Is an if missing? 
```suggestion
                                      Currently the memory footprint is only reduced if bp5 is used, HDF5 and bp4 does not support partial data dump.
```

Reply:
Discussed *offline* with @psychocoderHPC: the typo will be fixed in a followup pll request.